.github/workflows/maven.yml

```diff
@@ -19,15 +19,15 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up JDK 17
+    - name: Set up JDK 21
       uses: actions/setup-java@v3
       with:
-        java-version: '17'
-        distribution: 'temurin'
+        java-version: '21'
+        distribution: 'corretto'
         cache: 'maven'
     - name: Ensure code is formatted
-      run: mvn -B -Dstyle.color=always com.spotify.fmt:fmt-maven-plugin:check --file kaldb/pom.xml
+      run: mvn -B -Dstyle.color=always spotless:check --file kaldb/pom.xml
```

Comment:
Looks like `fmt-mvn-plugin` just got an update this morning. What do you think about giving this a shot? https://github.com/spotify/fmt-maven-plugin/pull/179

Reply:
that seems to do the job!